openshift/resultsdb-updater-test-template.yaml

```diff
@@ -23,7 +25,7 @@ parameters:
 - name: RESULTSDB_API_URL
   displayname: Resultsdb API URL
   description: API URL for Resultsdb
-  value: "http://resultsdb-test-${TEST_ID}-internal-api.myproject.svc:5001/api/v2.0"
+  value: "http://resultsdb-test-123-internal-api.myproject.svc:5001/api/v2.0"
```

Comment:
What is the purpose of having `123` in the URL? Why is it now hard-coded instead of using `${TEST_ID}`?

Reply:
The reason this was changed, b/c `TEST_ID` is also a parameter, and `oc process` is not able to resolve this value, when default values for other parameters are defined. `123` was chosen as this is the ID used in the examples given at the beginning of the templates.

Although, giving some thought, it might be sensible not to define a default value for `RESULTSDB_API_URL` (so that the template does not make any assumption about how resultsdb was deployed). What do you think?

Reply:
I agree. I think removing the default would be better.